yarn.lock

```diff
@@ -725,10 +725,10 @@
   resolved "https://registry.yarnpkg.com/@influxdata/flux/-/flux-0.5.1.tgz#e39e7a7af9163fc9494422c8fed77f3ae1b68f56"
   integrity sha512-GHlkXBhSdJ2m56JzDkbnKPAqLj3/lexPooacu14AWTO4f2sDGLmzM7r0AxgdtU1M2x7EXNBwgGOI5EOAdN6mkw==
 
-"@influxdata/giraffe@^2.13.1":
-  version "2.13.1"
-  resolved "https://registry.yarnpkg.com/@influxdata/giraffe/-/giraffe-2.13.1.tgz#bc4cfcab19082122a12c80182fbe07bc92f8bcb0"
-  integrity sha512-lkwSGfeJHkCzi4ptngi3NqA+fREMjJdeKCNUC6GxTKg4j5anODLKjwW4SSEDFK68A0AegawuUkY/DIjuMwBZ5A==
+"@influxdata/giraffe@^2.14.0":
+  version "2.14.0"
```

Comment:
i think Timmy merged in this change yesterday, so it might be worth merging master